code/game/gamemodes/cult/runes.dm

```diff
@@ -765,10 +786,15 @@ var/list/sacrificed = list()
 
 /////////////////////////////////////////SEVENTEENTH RUNE
 
-/obj/effect/rune/proc/wall(var/mob/living/user)
+/obj/effect/rune/proc/wall(var/mob/living/carbon/human/user)
 	user.say("Khari[pick("'","`")]d! Eske'te tannin!")
 	src.density = !src.density
-	user.take_organ_damage(2, 0)
+	if(ishuman(user))

```

Comment:
What about non-humantypes?

Reply:
This block of logic is so common it might also be a good idea to make it a proc.